### PR TITLE
Create Blog “creating-dscc-block-storage-resources-using-dscc-ansible-playbooks”

### PR DESCRIPTION
**Pull request by madhav-bharadwaj**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md
Comment (choukden) [inline content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md:9]: ...blog storage resources use case:
(type "Use case" --> "use case".)

File: content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md
Comment (choukden) [inline content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md:54]: Replace ;' with a colon for the sentence: This means that the creation of host requires two calls: one to get the initiator and another to create a host.

File: content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md
Comment (choukden) [inline content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md:106]: Make sure to escape the special character "_" for the variable names.

For the device type, shouldn't you add anything about Alletra 9K/Alletra 6K in addition to Primera and Nimble?

I might be wrong here, but I believe that Device type 1 relates to HPE Primera/HPE Alletra 9K storage device, and Device type 2 relates to HPE Nimble/HPE Alletra 6K storage devices.
You may want to double check with your team.

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Madhav,
Just a few comments for you to take into account for this post.
Let me know when the comments are incorporated and I will ask Dale Rensing to go through the Editorial review.

Best regards,
Denis

File: content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md
Comment (madhav-bharadwaj) [reply content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md:9]: This is supposed to be a sub heading. I have made sure the "Use case" is presented on a new line. So will leave the casing as is.

**Comment by madhav-bharadwaj** [comment]
@choukden , Thanks for your suggestions. I've incorporated most of your suggestions. Can you please have a look.

**Comment by madhav-bharadwaj** [comment]
> Hi Madhav,
> 
> Thank you very much for incorporating the changes. This looks good to me.
> 
> For the cross references of the blog posts, the anticipated blog link URL are as follows (should you want to cross reference the posts):
> 
> **Introducing Python SDK for Data Services Cloud Console** https://developer.hpe.com/blog/introducing-python-sdk-for-dscc/
> 
> **Automating operations on Data Services Cloud Console using Ansible Playbooks** https://developer.hpe.com/blog/automating-operations-on-dscc-using-ansible-playbooks/
> 
> **Creating Data Services Cloud Console Block storage resources using Ansible playbooks** https://developer.hpe.com/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks/
> 
> Dale is currently busy with higher priority activity in OCTO BU. This might delay the editorial review for your blog posts for a couple of days.
> 
> Thank you for your understanding,
> 
> Best regards, Denis

No issues with the delay. We have the storage announcement on 4th of April. It would be great to have the blogs public by then. Also, we have done a round of review by Dale. Can see her comments on the word draft that we used and I've incorporated most of her suggestions. So might speed things up when she gets to the review phase.

File: content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md
Comment (drensing) [inline content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md:105]: This should be "Creation of volume set"

File: content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md
Comment (drensing) [inline content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md:107]: Instead of saying "which is the kind of workload that is...", what would you think about changing it to "which indicates what kind of workload is required, like Oracle Database)." It would make more sense to me, but your call.

File: content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md
Comment (drensing) [inline content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md:129]: "Creation of volume (Primera)"

File: content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md
Comment (drensing) [inline content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md:173]: "Host Group" should be "host group"

File: content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md
Comment (drensing) [inline content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md:192]: Here, I would add "HPE GreenLake" prior to the first mention of "Data Services Cloud Console." We often  call out the full name of a product in the summary.

**Review by drensing — COMMENTED** [review]
Nice summary on this one, by the way.
Just a few items.
All the best,
Dale

File: content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md
Comment (madhav-bharadwaj) [reply content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md]: Changed it. Thanks.,

**Comment by madhav-bharadwaj** [comment]
@drensing , @choukden , I've incorporated all your suggestions. Can you please have a look?

**Pull request merged**
